shared/macros-ansible.jinja

```diff
@@ -141,6 +141,26 @@
 {{{ ansible_set_config_file(msg, "/etc/ssh/sshd_config", parameter, value=value, create="yes", prefix_regex='(?i)^\s*', validate="/usr/sbin/sshd -t -f %s", insert_before="^[#\s]*Match") }}}
 {{%- endmacro %}}
 
+{{#
+  High level macro to set a value in a shell-related file that contains var assignments. This
+  takes these values: msg (the name for the Ansible task), path to the file, a parameter to set
+  in the configuration file, and the value to set it to. We specify a case
+  sensitive comparison in the prefix since this is used to deduplicate since
```

Comment:
I think there is something missing at the end of the line.

Where is the case sensitive comparison done?

Reply:
This was a copy-paste error.